tests/test_integration.py

```diff
@@ -18,22 +18,31 @@
 
 
 def make_factory():
-    automap = automap_base()
+    # NOTE: Due to the changes made in the production database and the switch
+    # to using views instead of tables, we have attempted to reconstruct the
+    # schema with our test data.  It is an approximation of what is actually
+    # in production, but we are focused more on ensuring that the behavior and
+    # processing of our scripts and business logic are sound as opposed to
+    # mirroring the data exactly as it is at this time.
+    metadata = sa.MetaData(schema='disclosure')
+    automap = automap_base(bind=db.engine, metadata=metadata)
     automap.prepare(db.engine, reflect=True)
 
     class SchedAFactory(SQLAlchemyModelFactory):
         class Meta:
             sqlalchemy_session = db.session
-            model = automap.classes.fec_vsum_sched_a
-        filing_form = '11'
+            model = automap.classes.nml_sched_a
```

Comment:
We're going to follow up with this question around `disclosure.v_sum_and_det_sum_report` with @PaulClark2 and @jwchumley to get clarification.  If we do need to support this, we'll do it in a new PR.